shift/__manifest__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Elouan Le Bars <elouan@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
```suggestion
# Copyright 2020-2023 Coop IT Easy SC
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Removing Elouan from here, because copyright belongs to Coop IT Easy SC.